host/3.0/bionic/arm32v7/dotnet/dotnet.Dockerfile

```diff
@@ -22,18 +22,28 @@ RUN EXTENSION_BUNDLE_VERSION=1.8.1 && \
     mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
     unzip /$EXTENSION_BUNDLE_FILENAME -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION && \
     rm -f /$EXTENSION_BUNDLE_FILENAME && \
+    rm -r /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/${EXTENSION_BUNDLE_VERSION}/bin && \
     EXTENSION_BUNDLE_VERSION_V2=2.19.0 && \
     EXTENSION_BUNDLE_FILENAME_V2=Microsoft.Azure.Functions.ExtensionBundle.${EXTENSION_BUNDLE_VERSION_V2}_linux-x64.zip && \
     wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2/$EXTENSION_BUNDLE_FILENAME_V2 && \
     mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V2 && \
```

Comment:
from offline discussion - please see if we can delete V2 all together

Reply:
This PR does delete netcore 2.0 binaries. The bin directory contains the v2 binaries. I have deleted it from all bundles

Reply:
The unzipped content of the bundle zip files ends up in the final image. It contains binaries and template files that are referenced by the function runtime based on customer's host.json configuration.